### DISCUSSION TITLE
Signet install questions

### DISCUSSION THREAD

**Discussion by humansinstitute** [stackernews/stacker.news#259] [General]
Hi - I'm looking at installing a play version of stackernews on signet to have somewhere to test LNURL Auth, WebLN pay / receive. 

Pretty quick to get the local host version up and running, but I wanted to check am I correct in thinking that I'll need a local LND node to connect to, or is it possible to use a remote LND node?

**Comment by huumn** [comment]
You should be able to use a remote lnd node. Prod uses a remote node and the only difference between this and local dev is the node's host really. It's mostly a matter of making your remote node reachable from wherever your SN server is located (docker or otherwise).

Also, you'll want the url in the LNURL reachable from wherever you expect to interact with it which might require a http tunnel (e.g. ngrok).

**Comment by ekzyis** [comment]
BTW, there is also documentation about how to setup local LND now: https://github.com/stackernews/stacker.news/blob/master/docs/local-lnd.md
